pubspec.yaml

```diff
@@ -40,9 +40,19 @@ dev_dependencies:
   riverpod_lint: ^2.3.9
   test: ^1.24.9
   mockito: ^5.4.4
+  flutter_launcher_icons: "^0.13.1"
 
 flutter:
   uses-material-design: true
   assets:
     - assets/images/
     - assets/icons/
+
+# Command to generate the icons: `dart run flutter_launcher_icons`
+flutter_launcher_icons:
+  android: true
+  image_path: assets/images/proxima_white.jpg
+
+# Command to generate the splash screen: `dart run flutter_native_splash:create`
+flutter_native_splash:
+  color: "#FFFEFE"  # logo background color
```

Comment:
Shouldn't also set the `color` for the `android12:` ?

Reply:
No, this is if we need a special one. Currently, it works on android12 too.